language/go/config.go

```diff
@@ -369,7 +374,10 @@ func (*goLang) Configure(c *config.Config, rel string, f *rule.File) {
 				}
 				gc.preprocessTags()
 				gc.setBuildTags(d.Value)
-
+			case "go_proto":
+				if d.Value == "disable" {
```

Comment:
Let's `log.Printf` an error for any values we don't recognize.

Do you expect this will need to accept more than a Boolean value in the future? If not, lets use `strconv.ParseBool`. It seems like we'll only ever want to follow the current proto mode or disable rule generation, so a Boolean should be fine.